02/app.js

```diff
@@ -1,24 +1,32 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
 
-import List from './List';
-import Form from './Form';
+import List from "./List";
+import Form from "./Form";
 
 class App extends React.Component {
     state = {
         usersList: [],
-    }
+    };
+
+    addUser = (newUser) => {
+        this.setState((state) => {
+            return {
+                usersList: [...this.state.usersList, newUser],
```

Comment:
czyli tutaj można dować nowe id np. `usersList: [...this.state.usersList, {...newUser, id: uuid() }]`